astro.config.ts

```diff
@@ -31,11 +36,23 @@ export default defineConfig({
           label: "Tips",
           autogenerate: { directory: "tips" },
         },
+        {
+          label: 'Resources',
+          badge: 'New',
+          autogenerate: { directory: 'resources' },
+        },
       ],
       components: {
         PageTitle: "./src/components/starlight/PageTitle.astro",
         MarkdownContent: "./src/components/starlight/MarkdownContent.astro",
       },
     }),
   ],
+  vite: {
+    resolve: {
+      alias: {
+        '~': resolve(dirname(fileURLToPath(import.meta.url)), './src'),
+      },
+    },
+  }
```

Comment:
Why is this needed? Aren't we using something in the tsconfig?

Reply:
This is a change while I tried to fix the `astro check` issues. This would allow us to get rid of `"baseUrl": "."`, not sure if you care about that?

Reply:
Not at all! Just curious